Intersect.Server.Core/Database/GameData/GameContext.cs

```diff
@@ -72,6 +75,14 @@ public abstract partial class GameContext : IntersectDbContext<GameContext>, IGa
         //Time
         public DbSet<TimeBase> Time { get; set; }
 
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<ItemBase>()
+                .HasOne(item => item.EquipmentProperties)
+                .WithOne(prop => prop.Item)
+                .OnDelete(DeleteBehavior.Cascade);
```

Comment:
this should be reversed, it should be `Entity<EquipmentProperties>`, inverse the Has/With, and add `HasForeignKey<EquipmentProperties>(ep => ep.Descriptor)` after the `WithOne` before the `OnDelete`

Reply:
Done